src/includes/performance/connect-errors-spans/java.mdx

```diff
@@ -2,6 +2,16 @@
 
 Sentry errors can be linked with transactions and spans.
 
+<PlatformSection supported={["java"]} notSupported={["android", "java.spring", "java.spring-boot"]}>
+
+<Note>
+
+If you are using either our [Spring](/platforms/java/guides/spring/) or [Spring Boot](/platforms/java/guides/spring-boot/) integrations, errors are linked with spans automatically.
```

Comment:
Would be good to word this in a way that the information on this page is still useful. Users can still instrument their apps programatically while using the spring integration

Reply:
This is just about linking errors and spans. If you have better wording in mind, give it a go :-)

Reply:
```suggestion
We have dedicated performance features for our [Spring](/platforms/java/guides/spring/) and [Spring Boot](/platforms/java/guides/spring-boot/) integrations. In addition, errors are linked with spans automatically using either of these integrations.
```